test/message-input.test.js

```diff
@@ -106,4 +106,16 @@ describe('message-input', () => {
       expect(textArea.inputElement.getAttribute('aria-label')).to.be.equal('Viesti');
     });
   });
+
+  describe('disabled attribute propagation', () => {
```

Comment:
```suggestion
  describe('disabled property', () => {
```

We are testing the property so let's update the suite title accordingly.